apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts

```diff
@@ -486,6 +532,33 @@ export function InstructorAssessmentInstance({
                             `
                           : ''}
                       </td>
+                      <td class="text-right">
+                        <div class="dropdown js-question-actions">
+                          <button
+                            type="button"
+                            class="btn btn-secondary btn-xs dropdown-toggle"
+                            data-toggle="dropdown"
+                            aria-haspopup="true"
+                            aria-expanded="false"
+                          >
+                            Action <span class="caret"></span>
+                          </button>
+                          <div class="dropdown-menu dropdown-menu-right">
+                            ${resLocals.authz_data.has_course_instance_permission_edit
+                              ? html`
+                                  <button
+                                    class="dropdown-item"
+                                    data-toggle="modal"
+                                    data-target="#resetQuestionVariantsModal"
+                                    data-instance-question-id="<%= instance_question.id %>"
```

Comment:
Leftover EJS.

Reply:
Converted to a template literal expression in [cd0e458](https://github.com/PrairieLearn/PrairieLearn/pull/9578/commits/cd0e45818018ecb9941cf60705e037044a94db42).